script/smt-report

```diff
@@ -495,64 +432,22 @@ $LOG = SMT::Utils::openLog($logfile);
 # calculate best report
 my $cfg = SMT::Utils::getSMTConfig();
 
-if ( $cfg->val('NU', 'ApiType', 'NCC') eq 'NCC')
+my $api = $cfg->val('NU', 'ApiType', 'SCC');
```

Comment:
I don't think there is a problem with migration, but we use NCC as default everywhere else so this surprised me.